nuxeo-coldstorage-web/src/main/resources/web/nuxeo.war/ui/coldstorage/nuxeo-coldstorage.html

```diff
@@ -26,6 +27,14 @@
   </template>
 </nuxeo-slot-content>
 
+<nuxeo-slot-content name="restoreFromColdStorageDocumentContentAction" slot="DOCUMENT_ACTIONS" order="120">
+  <template>
+    <nuxeo-restore-content-from-coldstorage-button
+      document="[[document]]"
+    ></nuxeo-restore-content-from-coldstorage-button>
+  </template>
+</nuxeo-slot-content>
+
 <nuxeo-slot-content name="retrieveFromColdStorageDocumentContentAction" slot="BLOB_ACTIONS" order="120">
   <template>
     <nuxeo-retrieve-content-from-coldstorage-button
```

Comment:
@andreacornaglia could you change the order as we already have the same for the move action [Here](https://github.com/nuxeo/nuxeo-coldstorage/blob/master/nuxeo-coldstorage-web/nuxeo-coldstorage.html#L17-#L23)

Reply:
sure, in any case the two butttons will never display at the same time.